spec/controllers/tutors_controller_spec.rb

```diff
@@ -84,9 +69,22 @@
 
     describe 'GET #show' do
       before do
-        @tutor = create(:employed_tutor)
         user = User.new(role: 2)
         sign_in_auth(user)
+
+        affiliate = create(:affiliate)
+        @student = create(:student)
+        @tutor = create(:tutor)
+
+        create(:student) # not the same affiliate
+        s2 = create(:student) # already being tutored
+        t2 = create(:tutor) # to tutor s2
```

Comment:
`@` scopes it to the parent scope, meaning that it can then be used in sibling blocks. Generally if it's not needed in sibling blocks you shouldn't "dirty" the scope.